NSubstitute.DbConnection/NSubstitute.DbConnection/IMockQueryBuilder.cs

```diff
@@ -49,6 +49,32 @@ public interface IMockQueryBuilder
         /// <returns>The query builder</returns>
         IMockQueryBuilder WithOutputParameters(params (string Key, object OutputValue)[] parameters);
 
+        /// <summary>
+        /// Specifies that the query will only match if the given return parameters are passed.
+        /// The provided value will be set on the matching IDbCommand parameter.
+        /// </summary>
+        /// <param name="parameters">The parameters to match on</param>
+        /// <returns>The query builder</returns>
+        IMockQueryBuilder WithReturnParameters(IReadOnlyDictionary<string, object> parameters);
```

Comment:
I'm not 100% sure about this, but it looks as though you can only have one return parameter in a query?

Reply:
Oh of course yeah you can't have more than one in SQL, I assume the real API reflects that behavior too